src/components/ZclCreateModifyEndpoint.vue

```diff
@@ -58,7 +58,7 @@ limitations under the License.
             class="col v-step-2"
             use-input
             :multiple="enableMultipleDevice"
-            :use-chips="enableMultipleDevice"
+            use-chips
```

Comment:
What does this do?

Reply:
in standalone zigbee mode, we could not use the backspace / remove btn on the new card endpoint card. But if we modify this, it is working well. (I had a meeting with @paulr34, @dhchandw, @ethanzhouyc and this was one of the topics.)